flow/src/main/java/com/sequenceiq/flow/service/FlowService.java

```diff
@@ -195,9 +196,23 @@ public FlowCheckResponse getFlowState(String flowId) {
         flowCheckResponse.setFlowId(flowId);
         flowCheckResponse.setHasActiveFlow(!completed("Flow", flowId, List.of(), allByFlowIdOrderByCreatedDesc));
         flowCheckResponse.setLatestFlowFinalizedAndFailed(isFlowInFailedState(allByFlowIdOrderByCreatedDesc, failHandledEvents));
+        setStateEventFlowTypeOnFlowCheckResponse(flowCheckResponse, allByFlowIdOrderByCreatedDesc, "flow", flowId);
         return flowCheckResponse;
     }
 
+    public void setStateEventFlowTypeOnFlowCheckResponse(FlowCheckResponse flowCheckResponse, List<FlowLogWithoutPayload> relatedFlowLogs,
```

Comment:
why is this public?

Reply:
@lacikaaa Updated it to be private. Thank you!